src/scripts/includes/pythonscript.sh

```diff
@@ -1,3 +1,4 @@
+#Points out the envars and runs python in lind.
```

Comment:
I'm not sure this script is necessary as is. Something like this with a basic hello would be cool to add to the test suite, but that's a whole new build step.

We do need to keep track of these environment variables though. I think we for now should put them in a wiki somewhere. Eventually I'd like to add them as a default to Lind somewhere so we don't have to add them every time.

Reply:
For the pythonscript issue, we use this script for helloworld, flask and (coming soon) psycopg2 test. But yes, this is a temproary solution because we wouldn't like to export pythonhome-path every time we run a python script on lind.
Maybe we can alias `lind python [script]` as this.